pykonik/meetups/models.py

```diff
@@ -38,21 +38,35 @@ def get_upcoming(self, date=None):
             raise self.model.DoesNotExist
 
 
+class MeetupType(models.Model):
+    name = models.CharField(max_length=64)
+    has_agenda = models.BooleanField(default=True)
+
+    def __str__(self):
+        return self.name
+
+
 class Meetup(models.Model):
+    meetup_type = models.ForeignKey(MeetupType, null=True)
+    description = models.TextField(null=True, blank=True)
```

Comment:
I think you can remove null=True for char/text fields and update migrations, please see:

https://docs.djangoproject.com/en/1.11/ref/models/fields/#null

"Avoid using null on string-based fields such as CharField and TextField. If a string-based field has null=True, that means it has two possible values for “no data”: NULL, and the empty string. In most cases, it’s redundant to have two possible values for “no data;” the Django convention is to use the empty string, not NULL. One exception is when a CharField has both unique=True and blank=True set. In this situation, null=True is required to avoid unique constraint violations when saving multiple objects with blank values.

For both string-based and non-string-based fields, you will also need to set blank=True if you wish to permit empty values in forms, as the null parameter only affects database storage (see blank)."